### DISCUSSION TITLE
Is the `is_incomplete` note about Helix in the readme still an issue?

### DISCUSSION THREAD

**Discussion by sommerper** [Feel-ix-343/markdown-oxide#123] [Q&A] [answered]
The issue https://github.com/helix-editor/helix/issues/9797 seems to have been completed.

Although it speaks about being a temporary fix.

**Comment by Feel-ix-343** [comment, marked as answer]
Hey thanks for checking in! 

I have made a few PRs in attempts to fix this, but one of the maintainers is implementing it himself. So the issue is not resolved yet. Maintainers are super busy so this is totally understandable. 

We will certainly know it is resolved the issue is closed on the helix repo. 

Let me know if you have any other questions.
